src/platforms/common/configuration/options.mdx

```diff
@@ -577,13 +577,36 @@ Switches out the transport used to send events. How this works depends on the SD
 
 </ConfigKey>
 
-<ConfigKey name="http-proxy" supported={["node", "php", "python", "java", "dotnet", "rust"]}>
+<ConfigKey name="transport-options" supported={["javascript", "node"]}>
+
+Options used to configure the transport. This is an object with the following possible optional keys:
+
+<PlatformSection supported={["node"]}>
+
+- `headers`: An object containing headers to be sent with every request.
+- `proxy`: A proxy used for outbound requests. Can be http or https.
+- `caCerts`: A single or list of a path to a file or directory, or a buffer of CA certificates
+- `httpModule`: A custom HTTP module to use for requests. Defaults to the the native `http` and `https` modules.
+- `keepAlive`: Whether to keep the socket alive between requests. Defaults to `false`.
+
+</PlatformSection>
+
+<PlatformSection supported={["javascript"]}>
+
+- `headers`: An object containing headers to be sent with every request. Used by the SDK's fetch and XHR transports.
+- `fetchOptions`: An object containing options to be passed to the `fetch` call. Used by the SDK's fetch transport.
+
+</PlatformSection>
```

Comment:
This section is showing up for node.js and I assume it shouldn't? It's likely because the javascript is the fallback platform for node. Let's see if adding node to a notsupported list for this section will fix it.
```suggestion
<PlatformSection supported={["javascript"]} notSupported={["node"]}>

- `headers`: An object containing headers to be sent with every request. Used by the SDK's fetch and XHR transports.
- `fetchOptions`: An object containing options to be passed to the `fetch` call. Used by the SDK's fetch transport.

</PlatformSection>
```

Reply:
Yeah just noticed that as well, going to add this, lets see!